drake/systems/plants/RigidBodyTreeURDF.cpp

```diff
@@ -670,7 +697,7 @@ void parseRobot(RigidBodyTree * model, XMLElement* node, const map<string,string
   }
 }
 
-void parseURDF(RigidBodyTree * model, XMLDocument * xml_doc, map<string,string>& package_map, const string &root_dir, const DrakeJoint::FloatingBaseType floating_base_type)
+void parseURDF(RigidBodyTree * model, XMLDocument * xml_doc, map<string,string>& package_map, const string &root_dir, const DrakeJoint::FloatingBaseType floating_base_type, std::shared_ptr<RigidBodyFrame> weld_to_frame=nullptr)
```

Comment:
it would be nice to make the corresponding changes to the sdf parser, too.  but i guess not strictly required (it doesn't cause confusion in the interface about it being implemented).